.env.docker-compose

```diff
@@ -0,0 +1,14 @@
+# Used for the docker-compose file specifically for the database network configuraitions
+
+#DB Configuraitions.
```

Comment:
Please mention what are the things that needs to be changed and what need to constant.